### ISSUE TITLE
run without clubbtop limiter

### ISSUE THREAD

**Issue by adamrher** [NCAR/amwg_dev#108]
### Description of the run

CLUBB currently has a limiter that zeros out CLUBB tendencies above the chemical tropopause (and adds them to the active region). This is only done for tendencies of T, qv and CLDLIQ; all other tracers are diffused by CLUBB/KVH, no matter how high. We'd like to reassess the impact of this limiter w/ the new vertical grid. I think the new AMIP control is https://github.com/NCAR/amwg_dev/issues/105, and I'd like it to be identical to this run except for the removal of the limiter

### Suffix in the casename

noclubbtop

### Namelist modifications

same as https://github.com/NCAR/amwg_dev/issues/105

### Source modifications

same as https://github.com/NCAR/amwg_dev/issues/105, plus this w/ the limiter removed:
```
/glade/u/home/aherring/src/cam6_3_041.dtsens/usr_src/noclubbtop/clubb_intr.F90
```

### Sandbox

same as https://github.com/NCAR/amwg_dev/issues/105

### Contact info

@adamrher 
@JulioTBacmeister 
@cbardeen

### Any other relevant information

Enter relevant info

**Comment by adamrher** [comment]
@cecilehannay can you remind me what cam tag is used in https://github.com/NCAR/amwg_dev/issues/105?

**Comment by cecilehannay** [comment]
It used a cesm tag based on cam6_3_041:
/glade/work/hannay/cesm_tags/cesm2_3_beta08_cam6_3_041_MOM

**Comment by jedwards4b** [comment]
@adamrher, @cecilehannay  To be clear that is a branch and not an explicit tag.

**Comment by adamrher** [comment]
I think it would be good to mimic https://github.com/NCAR/amwg_dev/issues/105 b/c this is the only F compset run w/ a bug fix to ZM2 (albeit a SourceMod) and SE dycore. So lets proceed with this branch.

**Comment by jedwards4b** [comment]
@adamrher I believe that is already the case - you need to use a CESM tag as opposed to a cam tag to run a fully active compset.

**Comment by adamrher** [comment]
@cecilehannay any other questions about this run?

**Comment by cecilehannay** [comment]
I started the run (along with a new control) yesterday. The new control
uses the same config as the last control. #105 was a bit different. I will
update github with the new runs.
I am sorry for the delay in starting the runs. It was because we were
trying to add cosp in our development runs. But I have been having issues
so I decided to start these runs without cosp for now. I will sort out the
issues with Andrew and we will include cosp in the next run.
